### PR TITLE
libowfat 0.32

### PR DESCRIPTION
**Pull request by zachauten**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by mistydemeo** [comment]
Weird build failures:

```
io/iom_wait.c:79:12: error: use of undeclared identifier 'i'
      for (i=0; i<r; ++i) {
           ^
```

```
io/iom_wait.c:85:10: error: member reference base type 'struct kevent [128]' is not a structure or union
          *s=kev.ident;
             ~~~^~~~~~
```

**Comment by mistydemeo** [comment]
Aha - for the first one, a badly-written `#ifdef` put the variable declaration `int i` inside `#ifdef HAVE_EPOLL`. Will report.

**Comment by mistydemeo** [comment]
We can adapt OpenBSD's fix for the `kevent` issue: https://www.mail-archive.com/ports@openbsd.org/msg83004.html I did however also report that upstream.

**Comment by mistydemeo** [comment]
Looked at the upstream CVS repo, and there's already a fix. Here's a copy in my git mirror, which applies to 0.32: https://github.com/mistydemeo/libowfat/commit/278a675a6984e5c202eee9f7e36cda2ae5da658d

https://github.com/mistydemeo/libowfat/commit/278a675a6984e5c202eee9f7e36cda2ae5da658d.patch

File: Formula/libowfat.rb [hunk 1]
```diff
@@ -14,6 +14,11 @@ class Libowfat < Formula
     sha256 "be87e0da446834d6f8f808c434e854ff7c9eb88c3f899fc48a830b36117cac83" => :el_capitan
   end
 
+  patch do
+    url "https://github.com/mistydemeo/libowfat/commit/278a675a6984e5c202eee9f7e36cda2ae5da658d.patch?full_index=1"
+    sha256 "c3251de8ab374700b2123cd35f3406ebe3fca31a7c63b771e4ba683fe3d82b36"
```
Comment (Bo98) [inline Formula/libowfat.rb]:
```suggestion
    sha256 "32eab2348f495f483f7cd34ffd7543bd619f312b7094a4b55be9436af89dd341"
```

**Pull request closed without merging**